pkg/quota/admission/resourcequota/admission.go

```diff
@@ -4,21 +4,11 @@
 package resourcequota
```

Comment:
I think we should just remove this entire admission plugin.  I would create a filter on the admission plugin names which rewrites any "openshift.io/OriginResourceQuota" entry to "ResourceQuota" and de-dupes "ResourceQuota" with the last entry winning.

I'd also update the config validator to issue a warning if "openshift.io/OriginResourceQuota" appears in the config object and it should indicate that we'll remove toleration in the near future.